ts/Core/Renderer/BBoxObject.d.ts

```diff
@@ -28,6 +28,7 @@ export interface BBoxObject extends PositionObject, SizeObject {
     width: number;
     x: number;
     y: number;
+    poly?: { x: number, y: number }[]
```

Comment:
If we use tuples, we'll produce more terse code, and we'll be able to reuse existing functions from map geometry code.
```suggestion
    polygon?: [number, number][]
```